ert3/console/_console.py

```diff
@@ -225,6 +243,9 @@ def _main() -> None:
     if args.sub_cmd == "init":
         _init(args)
         return
+    elif args.sub_cmd == "service":
```

Comment:
Are you sure you do want this to be independent of the workspace? I would expect storage information to be persisted in `.ert` quite soon and then this would be violated...

Reply:
To initialise the workspace, ert-storage is being touched.

Reply:
Yes and by putting the `service` call right next `init` we are classifying it as should not require a workspace. I'm wondering if it would fit better further down with the actions that do require a workspace...

Reply:
I agree that it might make sense that the services will get "registered" in the workspace. However, this require a fix in the init of the workspace, which I do not believe is within the scope of this PR

Reply:
My bad! You are correct...